### DISCUSSION TITLE
Docker image

### DISCUSSION THREAD

**Discussion by stevefan1999-personal** [osuAkatsuki/bancho.py#448] [Ideas]
We need it so that I can build my helm chart, right now I'm not sure what image to use

**Comment by SysWhiteDev** [comment]
A pre-built Docker image for **bancho.py** is publicly available, the tag is: `osuakatsuki/bancho.py:latest`.
You can also find it on Docker Hub here: [https://hub.docker.com/r/osuakatsuki/bancho.py](https://hub.docker.com/r/osuakatsuki/bancho.py)
